src/WebApiTemplate/src/WebApiTemplate.App/Configuration/AkkaConfiguration.cs

```diff
@@ -208,8 +208,7 @@ public static HashCodeMessageExtractor CreateCounterMessageRouter()
             return o switch
             {
                 IWithCounterId counterId => counterId.CounterId,
-                ShardRegion.StartEntity startEntity => startEntity.EntityId,
-                _ => string.Empty
+                _ => null
```

Comment:
This will produce a `possible null value` warning during build until https://github.com/akkadotnet/akka.net/pull/7059 fix is released (1.5.16)